app/models/exam_registration.rb

```diff
@@ -73,6 +73,10 @@ def registered?(user)
     registrees.include? user
   end
 
+  def meets_criterion?(user)
```

Comment:
Is this method conflicting with delegate call above?

Reply:
well caught!